tests/integration/helpers.py

```diff
@@ -88,9 +90,16 @@ async def set_password(
 
 
 async def get_mongo_cmd(ops_test: OpsTest, unit_name: str):
-    ls_code, _, _ = await ops_test.juju(f"ssh --container {unit_name} ls /usr/bin/mongosh")
-
-    mongo_cmd = "/usr/bin/mongo" if ls_code != 0 else "/usr/bin/mongosh"
+    ls_code, _, stderr = await ops_test.juju(
+        f"ssh --container mongod {unit_name} ls /usr/bin/mongosh"
+    )
+    if ls_code != 0:
+        logger.error(f"mongosh not found. Reason: '{stderr}'")
+    # mongo_cmd = "/usr/bin/mongo" if ls_code != 0 else "/usr/bin/mongosh"
```

Comment:
For some reason this code `await ops_test.juju(
        f"ssh --container mongod {unit_name} ls /usr/bin/mongosh"
    )` 
 consistently fails with "ERROR unrecognized command: juju ssh --container mongod mongodb-k8s/0 ls /usr/bin/mongosh", but the command actually works in CLI. 
 
 Because we know that the code is used with MongoDB 6, for sake of this PR just return  "/usr/bin/mongosh"

Reply:
maybe we should replace `juju ssh` with `juju run`

Reply:
Let's have it in a separate PR